docs/prover/Report/CallResolutionTable.md

```diff
@@ -0,0 +1,56 @@
+CallResolutionTable
+=================
+
+The `CallResolutionTable` shows information about all the summarized calls in the program.
+It helps the user to better understand decisions made by the Prover- which calls in the rule code got inlined, which were replaced by summaries, and why.
```

Comment:
For clarity, The code verified by the Prover consists of several modules. However, by default, the Prover is aware of just one module. When the module is calling other modules, the Prover does not know how to identify them, let alone seeing and analyzing their code. In this case, the user guides the Prover by configuring it to identify the different modules and connect them through linking. This can be done through two ways:
1. Inlining- Taking the code of the two modules and building one big module with the code of both. 
2. Summaries- This is a mathematical description of the behavior of the module. It is usually short though less precise. It exists in two forms which are either be over approximating (describing more behaviors than there are actually implemented in the module) or under approximating (limiting the scope of what the other module can do).